docs/src/utils/reader.ts

```diff
@@ -15,11 +15,13 @@ export async function getNavigationMap() {
       const { discriminant, value } = link;
       const page = discriminant === 'page' && value ? pagesBySlug[value] : null;
       const url = discriminant === 'url' ? value : `/docs/${page?.slug}`;
+      const comingSoon = discriminant === 'coming-soon' ? true : false;
```

Comment:
You don't need the ternary part of this. `discriminant === 'coming-soon'` evaluates to `true` or `false`. As a general rule, if you find yourself adding a boolean literal in a ternary, it usually means you can simplify the code.

Also, because this variable isn't reused, I would suggest removing this line then making line 24: `comingSoon: discriminant === 'coming-soon'`.

Reply:
Ahh silly me! Thanks Laney, I was a bit quick on the gun there 😅